Gems/LyShine/Code/Source/UiCanvasComponent.h

```diff
@@ -230,12 +230,12 @@ class UiCanvasComponent
     void MarkRenderGraphDirty() override;
     // ~UiCanvasComponentImplementationInterface
 
-    // RenderToTextureRequests
+    // LyShine::RenderToTextureRequestBus::Handler
```

Comment:
Nitpick: For styling consistency this should be updated to `// LyShine::RenderToTextureRequestBus overrides ...`

Reply:
fixed!